src/main/java/com/monov/cloud/gateway/CloudGatewayApplication.java

```diff
@@ -2,7 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.client.loadbalancer.LoadBalanced;
 import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.client.RestTemplate;
 
 @EnableEurekaClient
```

Comment:
You probably no longer need this annotation since we decided to stop using the gateway as a router only. Also optimize your imports via Ctrl + Alt + O

Reply:
done